jedi/settings.py

```diff
@@ -143,6 +143,11 @@
 ``globals()`` modifications a lot.
 """
 
+allow_descriptor_getattr = True
+"""
+Controls whether descripters are evaulated
```

Comment:
2 typos: descriptors and evaluated.

Maybe use this sentence? `Controls whether descriptors are evaluated when using an Interpreter. This is something you might want to control when using Jedi from a Repl (e.g. IPython).`

Reply:
Thank you for the assistance on the help here, doc string updated